src/ck-ldb/BaseLB.h

```diff
@@ -127,11 +125,11 @@ class BaseLB: public CBase_BaseLB
     int getRecvHash(LDCommData &cData);
     void clearCommHash();
     void clear() {
-      n_objs = n_migrateobjs = n_comm = 0;
-      objData.free();
-      commData.free();
-      from_proc.free();
-      to_proc.free();
+      n_migrateobjs = 0;
+      objData.clear();
```

Comment:
Should we be calling shrink_to_fit() after each of these clear calls? Or do we expect these vectors to be reused after this with roughly the same size as before it?

Reply:
No, the expectation is that these should be reused with similarly sized data, so we should leave them as is.